package.json

```diff
@@ -7,11 +7,11 @@
     "url": "https://www.algolia.com"
   },
   "scripts": {
-    "dev": "babel-node index.js",
+    "dev": "babel-node src/index.js",
```

Comment:
Side question, do we still need babel-node you think? We might just want to avoid using import/export and go with require.

So we can drop babel* for this nodejs project. Using latest nodejs native will allow for better stack traces, which is way more important than being able to "use" (fake to) import. I know I did that but I want your opinion on it.

Reply:
We can use normal node now that it has rest/spread for objects!

<img width="290" alt="screen shot 2018-01-16 at 09 54 00" src="https://user-images.githubusercontent.com/6270048/34979975-3c052760-faa3-11e7-8709-fa57528c7115.png">

Reply:
@vvo I won't use require, since it's not worth it to change it now that import will be supported so soon in node. Babel is fine for now